sources/web/datalab/common.d.ts

```diff
@@ -87,6 +87,26 @@ declare module common {
      * If true, allow HTTP requests via websockets.
      */
     allowHttpOverWebsocket: boolean;
+    
+    /**
+     * Whether to automatically back up user's contents dir to GCS
+     */
+    enableAutoGCSBackups: boolean;
```

Comment:
By putting these inside of the settings.json file, they get tied to the source code and a user won't be able to override them.

Instead, these should go into the user settings and be read using the ['loadUserSettings'](https://github.com/googledatalab/datalab/blob/master/sources/web/datalab/settings.ts#L90) method.